crates/uv/tests/pip_compile.rs

```diff
@@ -629,13 +629,7 @@ dependencies = [
 "#,
     )?;
 
-    // In addition to the standard filters, remove the temporary directory from the snapshot.
-    let filters: Vec<_> = [(r"file://.*/", "file://[TEMP_DIR]/")]
-        .into_iter()
-        .chain(INSTA_FILTERS.to_vec())
-        .collect();
```

Comment:
Filters like these are entirely unnecessary and were handled by `context.filters()` before any changes here, but I believe it wasn't clear that `context.filters()` existed.